README.md

```diff
@@ -297,6 +297,10 @@ aws-adfs integrates with:
       --help          Show this message and exit.
     ```
 
+## Supported Devices
+
+* RSA Tokens are supported by duo security using the `Passcode` authentication method
+
```

Comment:
I'd rather put that information in the existing section: https://github.com/venth/aws-adfs/pull/356/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R19